src/warnAboutLinterEslintFixOnSave/index.js

```diff
@@ -1,14 +1,21 @@
 // @flow
-const { shouldUseEslint, isLinterEslintAutofixEnabled, addWarningNotification } = require('../atomInterface');
+const {
+  shouldUseEslint,
+  isLinterEslintAutofixEnabled,
+  addWarningNotification,
+  isPackageActive,
+} = require('../atomInterface');
 
 const MESSAGE =
   "prettier-atom: linter-eslint's `Fix on Save` feature is currently enabled. " +
   'We recommend disabling this feature when using the ESlint integration from prettier-atom';
 const OPTIONS = { dismissable: true };
 
+const isLinterEslintActive = () => isPackageActive('linter-eslint');
+
 const displayWarning = () => addWarningNotification(MESSAGE, OPTIONS);
 
 const warnAboutLinterEslintFixOnSave = () =>
-  shouldUseEslint() && isLinterEslintAutofixEnabled() && displayWarning();
+  isLinterEslintActive() && shouldUseEslint() && isLinterEslintAutofixEnabled() && displayWarning();
```

Comment:
let's keep the API simpler by having `atomInterface` just expose `isLinterEslintEslintAutofixEnabled`. This way we don't have to do more than one thing in this function. In addition, we will simply be able to mock out `isLinterEslintAutofixEnabled`, instead of having to mock the atom interface outside of the atomInterface tests.